### ISSUE TITLE
3080 on Linux just hangs when faces are loading

### ISSUE THREAD

**Issue by MotorCityCobra** [iperov/DeepFaceLab#5283]
I unplugged my 1080ti from my Linux and plugged in a 3080 and now the script will not start training. It freezes after the part where the faces from the src and the dst are done loading and before the model should start training and the preview box should open.

The program just hangs with no error as if it is getting ready to do something but it just sits.

I've tried with a few CUDA versions and an older Tensorflow version and the 2.4.0 Tensorflow version and if I remember correctly this hanging after the faces loading is always what happens.

Is there a straight forward remedy for using DeepFaceLab on Linux with a 3080?

**Comment by xwyf0505** [comment]
hi,  I face the simillar issue on GeForce RTX 2080 Ti.  it had be stuck  at the extaract from data_src to data_dst.  Have you solve it ？

**Comment by MotorCityCobra** [comment]
> Have you solve it ？

Nope. I was also having this exact same hang up when I was training with my both my 1080tis instead of just one. Though I have also tried training with one 3080 vs the all 3080s I have and it doesn't work.
Are you on Linux? I'm thinking of trying on Windows. I don't know what else to do here. Maybe there is a problem with my motherboard and CPU being five years old. I don't understand. I wish I got an error message instead of the freeze.

**Comment by MotorCityCobra** [comment]
I installed in Windows and it seems to be working. I'm not sure if it's using all of my 3080s, but it's using some of them.
This repo may just not work well with Linux. Like I said, I had problems with it when I was using 1080tis.
Get a 128 SSD, but Windows 10 on it and do a dual boot, bro.

**Comment by ethanchinaski** [comment]
I have that Problem on Windows, this is all i get
        
`Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : GeForce RTX 3080

[0] Which GPU indexes to choose? : 0
0

[wf] Face type ( f/wf/head ?:help ) : wf
wf
[0] Max number of faces from image ( ?:help ) :
0
[512] Image size ( 256-2048 ?:help ) : 256
256
[90] Jpeg quality ( 1-100 ?:help ) : 95
95
[n] Write debug images to aligned_debug? ( y/n ) : n
Extracting faces...
Running on GeForce RTX 3080
  0%|                                                                                                                     | 0/13154 [00:00<?, ?it/s]`

pressing space ends the program which hangs as long as i dont use any keys

**Comment by ogladkov** [comment]
Does tensorflow import in your environment well? I had that problem and found out that tf couldn't import properly. Try it manually in your env.

**Comment by ghost** [comment]
Did you ever find the answer? If so, would you mind sharing it and closing this issue?
